app/models/employment/AllEmploymentData.scala

```diff
@@ -25,4 +25,17 @@ case class AllEmploymentData(hmrcEmploymentData: Seq[EmploymentSource],
 
 object AllEmploymentData {
   implicit val format: OFormat[AllEmploymentData] = Json.format[AllEmploymentData]
+
+  def employmentIdExists(allEmploymentData: AllEmploymentData, employmentId: Option[String]): Boolean = {
+    if(employmentId.isDefined) {
+      val idExistsInHMRCData: Boolean = allEmploymentData.hmrcEmploymentData.exists(_.employmentId == employmentId.get)
+      val idExistsInCustomerData: Boolean = allEmploymentData.customerEmploymentData.exists(_.employmentId == employmentId.get)
+
+      if (idExistsInHMRCData || idExistsInCustomerData) true else false
+    } else {
+      false
+    }
```

Comment:
Think this can be simplified with this code
```suggestion
    employmentId.exists { employmentId =>
      val idExistsInHMRCData: Boolean = allEmploymentData.hmrcEmploymentData.exists(_.employmentId == employmentId)
      val idExistsInCustomerData: Boolean = allEmploymentData.customerEmploymentData.exists(_.employmentId == employmentId)

      idExistsInHMRCData || idExistsInCustomerData
    }
```